types/index.d.ts

```diff
@@ -0,0 +1,109 @@
+// Minimum TypeScript Version: 3.8
+/// <reference types="node" />
+import type { ConnectionOptions } from 'typeorm';
+import type { IncomingMessage, ServerResponse } from 'http';
+import type { PossibleProviders } from './providers';
+import Adapter from './adapters';
+
+interface InitOptions {
+  site: string;
```

Comment:
I can't find any information about this option from the [docs](https://next-auth.js.org/configuration/options), what is it for ?

Reply:
Good catch, I see this options have been simplified quite a lot on **v3**, I'll update on the [types PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45775/files)